chapters/equations.tex

```diff
@@ -793,6 +794,9 @@ \section{Initialization, initial equation, and initial algorithm}\label{initiali
 
 Before the start of the integration, it must be guaranteed that for all variables \lstinline!v!, \lstinline!v = pre(v)!.
 If this is not the case for some variables \lstinline!vi!, \lstinline!pre(vi) := vi! must be set and an event iteration at the initial time must follow, so the model is re-evaluated, until this condition is fulfilled.
+In details this means that during initialization initial equations and normal equations are solved with \lstinline!v! and \lstinline!pre(v)! as unknowns without any event iterations.
+Then \lstinline!v! is copied to \lstinline!pre(v)! and only the normal equations are solved repeatedly until \lstinline!v = pre(v)!.
+Tools may optimize this to not compute unnecessary \lstinline!pre(v)! and only perform the event iteration if necessary.
```

Comment:
What is the purpose of writing out this possibility of optimization normatively?

Reply:
Well, given that some only want to standardize the result before starting the continuous integration I think it is good to have this.

Reply:
OK, while I can understand the attempt to address these parties, it still looks odd to me to say such a thing normatively.  Hence:
```suggestion

\begin{nonnormative}
Tools may optimize this by not computing unnecessary \lstinline!pre(v)!, and only perform the event iteration if necessary.
\end{nonnormative}
```